### DISCUSSION TITLE
Kyuubi on SSL Security support

### DISCUSSION THREAD

**Discussion by kmsantanu** [apache/kyuubi#928] [Q&A]
HOw to support Kyuubi on SSL Security

**Comment by yaooqinn** [comment]
https://kyuubi.apache.org/docs/latest/deployment/settings.html#authentication

**Comment by kmsantanu** [comment]
Hello

We created keystore and truststore. Which parameter we can configure it.

Did it support ssl directly?

We want to do ssl security to kyubi to do  connect ssl enabled cassandra
database
Please guide us
Thanks
San

**Reply by yaooqinn** [reply]
these functionalities are still missing in Kyuubi
